Jenkinsfile

```diff
@@ -0,0 +1,56 @@
+@Library('mdblp-library') _
+pipeline {
+    agent any
+    stages {
+        stage('Initialization') {
+            steps {
+                script {
+                    utils.initPipeline()
+                    if (env.GIT_COMMIT == null) {
+                        // git commit id must be a 40 characters length string (lower case or digits)
+                        env.GIT_COMMIT = "f".multiply(40)
```

Comment:
out of curiosity, why do we need a fake GIT_COMMIT?

Reply:
Ask @ydrisR about this.

Reply:
For local pipeline development without git, we need a commit id at least in publish scripts (and we don't have it with a local pipeline triggered manually).

Reply:
thanks @CedricPerotto